pkg/minikube/config/config.go

```diff
@@ -48,6 +49,10 @@ const (
 	ShowDriverDeprecationNotification = "ShowDriverDeprecationNotification"
 	// ShowBootstrapperDeprecationNotification is the key for ShowBootstrapperDeprecationNotification
 	ShowBootstrapperDeprecationNotification = "ShowBootstrapperDeprecationNotification"
+	// TimeFormat is the format that should be used when outputing time
+	TimeFormat = time.RFC1123
+	// User represents the key for the global user parameter
```

Comment:
key in what? Also, mention what this parameter is used for (it's not obvious)

Reply:
Updated comment to make it more descriptive